kamon-logback-stackdriver/src/main/scala/kamon/stackdriver/StackdriverMarker.scala

```diff
@@ -0,0 +1,80 @@
+package kamon.stackdriver
+
+import java.util
+
+import kamon.stackdriver.StackdriverMarker._
+import org.slf4j.Marker
+
+import scala.collection.concurrent.TrieMap
+import scala.collection.JavaConverters._
+
+abstract class StackdriverMarker(implicit enc: LogValue.Writer) extends Marker {
```

Comment:
I'm curious if there is some testsuite which could be launched against this Marker implementation to ensure it behaves as it should.

Reply:
Yeah `BasicStackdriverMarkerTest`, against `case class BasicStackdriverMarker(name: String, value: LogValue) extends StackdriverMarker`

Reply:
https://github.com/qos-ch/slf4j/blob/master/slf4j-api/src/test/java/org/slf4j/BasicMarkerTest.java